keps/sig-architecture/3935-oldest-node-newest-control-plane/README.md

```diff
@@ -0,0 +1,796 @@
+
+    section with n-2
+    Qualify v1.40                                     : n2_140qual,        2030-04-01,4w
+    Run v1.40                                         : n2_140run,         after n2_140qual, 1y
+    Qualify v1.41/v1.40                               : n3_143qual1,       2031-04-01,4w
+    Qualify v1.42/v1.40                               : n3_143qual2,       2031-04-01,4w
+    Qualify v1.42                                     : n3_143qual3,       2031-04-01,4w
+    Qualify v1.43/v1.42                               : n3_143qual4,       2031-04-01,4w
+    Qualify v1.43                                     : n2_143qual,        2031-04-01,4w
+    Control Plane → v1.41 → v1.42                     : n2_142cpupgrade,   after n2_143qual, 1d
+    Nodes → v1.42 (drain/recreate workloads)          : n2_142nodeupgrade, after n2_142cpupgrade,  1w
+    Control Plane → v1.43                             : n2_143cpupgrade,   after n2_142nodeupgrade,  1d
+    Nodes → v1.43 (drain/recreate workloads)          : n2_143nodeupgrade, after n2_143cpupgrade,  1w
+    Run v1.43...                                      : n2_143run,         after n2_143nodeupgrade, 2031-07-01
+```
+-->
+
+There is nothing magical about the number 2 in the existing version skew policy.
+That policy intended to allow the oldest supported node to work with the newest supported control plane (exactly what this KEP is trying to accomplish).
+When there are 3 supported minor versions, max skew of n-2 appears to make sense.
+
+However, as part of the [yearly support period](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1498-kubernetes-yearly-support-period#proposal) KEP,
+it became clear that overlap between the oldest and newest version is required to actually give users a version they can use for a year.
+This led to the current policy of supporting each minor version for approximately 14 months, giving users a 2-month period to qualify and upgrade following a new Kubernetes release
+before the oldest supported minor version reaches end-of-life. During that 2-month period, there are actually 4 supported minor versions.
+
+Updating the skew policy to allow up to n-3 nodes brings the policy back in line with its original goals.
+
+### Goals
+
+* Allow users to make full use of the [yearly support period](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1498-kubernetes-yearly-support-period)
+* Streamline upgrade paths for nodes / node pools to minimize required workload churn
+* Validate oldest supported node components work with latest control plane components
+* Update the [Kubernetes version skew policy](https://kubernetes.io/releases/version-skew-policy/) to indicate node components from the oldest supported minor version work with control plane components from the newest supported minor version, starting with Kubernetes `v1.25` nodes and `v1.28` control planes.
```

Comment:
Do we want to add "Identify additional testing needed going forward"?

Reply:
I can... I'm a little unsure whether that belongs in this KEP or in an issue tracking test improvements we already want for our existing versions

Reply:
And also should we update the PRR template so KEP authors are reminded that about the node/control-plane version skew policies?